index.bs

```diff
@@ -1040,6 +1185,111 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
       {{Screen}}.
     </p>
   </section>
+
+  <section>
+    <h3 id='video-display-capabilities'>Video Display Capabilities</h3>
+
+    <section>
+      <h4 id='video-display-configuration'>VideoDisplayConfiguration</h4>
+
+      <pre class='idl'>
+        dictionary VideoDisplayConfiguration {
+          required unsigned long width;
+          required unsigned long height;
+          required boolean hasHdrCapabilities;
+        };
+      </pre>
+
+      <p>
+        The <dfn for='VideoDisplayConfiguration' dict-member>width</dfn> member
+        represents the width of the display in physical pixels, not CSS pixels.
```

Comment:
This seems not quite right. The important thing we're trying to capture here is that these are the dimensions of the video plane (ie not the graphics plane). I think we still want to measure those dimensions in CSS pixels rather than physical device pixels. @mounirlamouri @hober - double check me

Reply:
The main use-case for this is to decide whether it is worth fetching 4K or not. Don't we want physical pixels for that ?

Reply:
I follow you, but I wonder if we should expose it in that way. My worry is whether if its semantically consistent for high density displays. 

As I understand the goal, we want to signal when a device's video plane is a different resolution from its graphics plane. For devices where the planes match (everywhere except TVs / dongles), I was thinking window.screen.height should = window.screen.video.height (same for width). Would you agree? If we use device pixels for video the two will not match for displays with device-pixel-ratio > 1. I worry this introduces inconsistent layout semantics.

For example, take a pixel-doubled 4K retina dispaly. The bottom right corner in terms of physical pixels is at 4096,2304, but in terms of CSS px you really only half that area to layout your page. If you try to size a video using CSS from 0,0 to 4096,2304 thinking you're filling the screen, you will actually be viewing only the top left quadrant.

Now take a TV with a device-pixel-ratio=1 and with video plane dimensions matching its native resolution of 4096x2304. Say its graphics plane is only half that. Again, try using html to size a video from 0,0 to 4096,2304 - this time (if I understand how these planes work) I expect the video is actually filling the screen? 

Do I have that last part right? I'm not sure I understand how folks do layout for a device with different planes. If I have understood it correctly, it would mean mean screen.video has different semantics in different settings. If we instead use CSS px (as a unit, not referring to a plane) the values would represent a layout area for both settings.

Reply:
@chcunningham @mwatson2 can we please move this discussion here: https://github.com/w3c/media-capabilities/issues/133 so that we can easily find the agreement in the future